funcx_sdk/funcx/sdk/executor.py

```diff
@@ -161,21 +161,21 @@ def submit(self, function, *args, endpoint_id=None, container_uuid=None, **kwarg
                 self._function_registry[function] = function_id
                 log.debug(f"Function registered with id:{function_id}")
 
-        task_id = self._task_counter
-        self._task_counter += 1
+        future_id = self._future_counter
+        self._future_counter += 1
 
         assert endpoint_id is not None, "endpoint_id key-word argument must be set"
 
         msg = {
-            "task_id": task_id,
+            "future_id": future_id,
             "function_id": self._function_registry[function],
             "endpoint_id": endpoint_id,
             "args": args,
             "kwargs": kwargs,
         }
```

Comment:
I'm going to do a follow-up to swap out this dict for a NamedTuple or dataclass or something, to see if that helps this even more.

I was concerned for a minute that this was sending `"future_id"` up to the service and that I would need to look and make sure that there was nothing doing `log.info("got task %s", request["task_id"])` (-> 💥 ).